apps/omg_watcher/lib/supervisor.ex

```diff
@@ -100,13 +100,11 @@ defmodule OMG.Watcher.Supervisor do
         synced_height_update_key: :last_convenience_exit_processor_eth_height,
         get_events_callback: &Eth.RootChain.get_standard_exits/2,
         process_events_callback: fn exits ->
-          exits =
-            Enum.map(exits, fn %{exit_id: exit_id} ->
-              {:ok, {_, _, _, position}} = OMG.Eth.RootChain.get_standard_exit(exit_id)
-              OMG.API.Utxo.Position.decode(position)
-            end)
+          _ =
```

Comment:
Is it ok that we ignore if those inserts failed?

Reply:
success of a single insert is checked within. I'll rename these to `xyz!` and get rid of this `_ = ` here, it's misleading